testing/binding.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: brigade-cr
+rules:
+  - apiGroups:
+      - '*'
+    resources:
+      - '*'
+    verbs:
+      - '*'
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: brigade-crb
+  namespace: default
+subjects:
+- kind: ServiceAccount 
+  namespace: default
+  name: brigade-worker
+roleRef:
```

Comment:
As mentioned above, just rolebind against `cluster-admin`

Reply:
I have replaced it with cluster-admin in latest push thank you.